NewUser'sGuide.md

```diff
@@ -0,0 +1,250 @@
+
+**Global options**
+
+Each of the options has already been explained in the **Manual Game Installation** section, however the difference is that each of the options here affects all of the games in your Lutris library, unless a game’s Runner options has been specifically changed.
+
+## Context Menu
+
+Right-clicking on any installed game in Lutris, provides the user with these options:
+- **Play** – runs the game.
+- **Show Logs** - opens a separate window and is useful for finding out why particular games are not running.
+- **Configure** - allows you to edit different settings of the game, as described in the **Manual Game Installation** section.
+
+Additionally, images (like a game’s artwork) can also be placed here for as a custom banner and icon, just click the boxes with the Lutris logo inside and choose your image. To remove any images, click the “brush” icon.
+
+A game’s **Identifier** and **Directory** can also be changed here, useful if you decide to move the game’s files to another location.
+
+- **Add to favorites** - adds the game to the **Favorites** submenu on the left-hand side of the client under **Library**.
+- **Browse files** - opens the directory where the game is installed in your chosen file manager.
+- **Create desktop shortcut** - creates a nice shortcut on your desktop.
+- **Create application menu shortcut** - places a shortcut for the game in your system’s application menu.
+- **Install another version** - install a different version of the game through this option.
+- **Remove** - this removes the game as Installed from your Library (including associated Play Time and other meta data), but does not delete any of the game’s files.
+- **View on Lutris.net** - shows additional information about the game on [Lutris.net](https://lutris.net/) as well as if any community installers are available for it, what platform it is for, release date and more.
+- **Hide game from library** - hides games you don’t want to see in your library. To view the game again, go to the Lutris Settings Menu and click “Show Hidden Games”. If you want to unhide the game permanently, follow the previous steps and right-click the game and select “Unhide game from library”.
+
+
+## Navigating Your Library
+
+### Library
+
+**Games**
+
+Under Library on the left-hand side, clicking **Games** will display all of the available games that are currently installed in your library.
+
+**Recent**
+
+Clicking **Recent** will display all of the games that you have played recently.
+
+**Favorites**
+
+Clicking **Favorites** shows you every game you have selected as your favourites, either via right-clicking the game in your library and selecting “Add to favorites” or the drop-down menu next to the game’s “Play” button at the bottom of the client and then selecting “Add to favorites”.
+
+## Sources
```

Comment:
The sources were mentioned before, I think it would be nice to only have them mentioned once in the guide.

Reply:
Okay, I have moved the section discussing the Sources to the top of the file. Now, under **Installing a Game**, the next section down is **Configuring your Sources** and the next one down after that is **Installation of a Game from a Source**
```suggestion
## Installing a Game

In Lutris, there are many ways to install games from your game collections. For this guide, it will only concentrate on installing a game from a Source.

### Configuring your Sources

Under **Sources** on the left-hand side of the client, you’ll find a list of all the Sources you currently have enabled. Clicking one shows all the games that are available from that particular Source.

**Lutris Source**

The **Lutris** source displays all of the games you have registered in your [Lutris.net](https://lutris.net/) library and allows you to search for any community installers that are available for a particular game. Both of these options are placed in tabs at the top of the client after selecting the source.

**Local Source**

The **Local** source displays all of your native Linux games.

**GOG Source**

The **GOG** source lists every game in your [GOG](https://www.gog.com/) library.

**Humble Bundle Source**

The **Humble Bundle** source shows all the games in your [Humble Bundle](https://www.humblebundle.com/) library.

**Epic Games Store Source**

The **Epic Games Store** source will display every game in your [Epic Games Store](https://www.epicgames.com/store/en-US/) library.

**Note – for GOG, Humble Bundle and the Epic Games Store, you have to log into the appropriate source via the connect button that appears after hovering over the source itself, otherwise none of the games linked with the associated source are displayed.**

**Steam**

Once you have the native Linux Steam client installed on your distribution and logged in successfully, all of your Steam games (regardless of platform), will appear here.

**Steam for Windows Source**

The Steam for Windows source lists every Steam game available in your library, run via the Wine Steam Runner. This option is available, in case you do not wish to use the native Linux Steam client with Proton.

**The Lutris team [has said](https://twitter.com/LutrisGaming/status/1454539182647377920) that integration for Steam for Windows will be removed, whilst leaving the Wine Steam Runner for those who still wish to use it.**

### Installation of a Game from a Source

Installing games from a Source is usually straightforward with Lutris.

1. Select a Source, then the game you wish to play and click "Install" at the bottom of the client. 
2. Lutris will then open a dialogue box and offer you some options for installation (downloading the base game, additional audio files, mods and much more). Depending on the Source used, the options presented will vary.
3. Once you click the "Install" button in the dialogue box, Lutris will handle the rest for you. For example. this includes downloading and running the installer for GOG games (with any associated community written installers for additional configuration) or for a Steam game, opening your Steam client, downloading the game and closing the client when the installation is done.
```